build-coordinator/src/main/java/org/jboss/pnc/core/builder/BuildCoordinator.java

```diff
@@ -153,25 +180,17 @@ private ProductMilestone getProductMilestone(BuildConfigurationSet buildConfigSe
         return buildConfigSet.getProductVersion().getCurrentProductMilestone();
     }
 
-    private void build(BuildSetTask buildSetTask, User userTriggeredBuild) throws CoreException {
+    private void build(BuildSetTask buildSetTask) throws CoreException {
 
-        BuildTasksTree buildTasksTree = new BuildTasksTree(
-                this,
-                buildSetTask,
-                userTriggeredBuild,
-                () -> datastoreAdapter.getNextBuildRecordId());
-
-        Predicate<Vertex<BuildTask>> acceptOnlyStatus = (vertex) -> {
```

Comment:
If I remember correctly this comes from JBoss commons dependency. Can we remove it?